src/state/multicall/hooks.ts

```diff
@@ -99,7 +102,7 @@ function useCallsData(calls: (Call | undefined)[], options?: ListenerOptions): C
         ? calls.map<CallResult>(call => {
             if (!chainId || !call) return INVALID_RESULT
 
-            const result = callResults[chainId]?.[toCallKey(call)]
+            const result = callResults?.[toCallKey(call)]
```

Comment:
Please explain more why do this?

Reply:
I move `.[chainId]` on to `useSelector` above, so this line I remove `[chainId]`